src/test/resources/com/sap/sgs/phosphor/fosstars/nvd/NVD_part.json

```diff
@@ -6,253 +6,713 @@
   "CVE_data_timestamp": "2020-04-02T08:25Z",
   "CVE_Items": [
     {
-      "cve" : {
```

Comment:
Did you only update formatting in this file? If so, let's please revert it.

Reply:
I will revert this file as this was to support a test case. If that is not required. I will revert this.